main.py

```diff
@@ -971,6 +1053,72 @@ def suggest(limit=10):
     return better_jsonify(suggestions=suggestions)
 
 
+@app.route("/locations", methods=["GET"])
+@max_age(seconds=60 * 5)
+def locations():
+    """ Render locations page """
+    kind = request.args.get("kind")
+    period = request.args.get("period")
+
+    days = _TOP_LOCATIONS_PERIOD
+    if period == "week":
+        days = 7
+
+    kind = kind if kind in LOCATION_TAXONOMY else None
+
+    with SessionContext(commit=False) as session:
+        locs = top_locations(enclosing_session=session, kind=kind, days=days)
+
+    return render_template("locations/locations.html", locations=locs, period=period)
```

Comment:
Is a path with forward slashes definitely allowed under Windows?

Reply:
Yes, they are apparently processed by Jinja and not treated naively as filesystem paths.